src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1065,8 +1080,12 @@ class DocBaseViewer extends BaseViewer {
             );
         }
 
-        this.controls.add(__('zoom_out'), this.zoomOut, 'bp-doc-zoom-out-icon', ICON_ZOOM_OUT);
-        this.controls.add(__('zoom_in'), this.zoomIn, 'bp-doc-zoom-in-icon', ICON_ZOOM_IN);
+        this.zoomControls.add(this.pdfViewer.currentScale, {
```

Comment:
I think the transparency of this API could be better. It's not clear to me what the result is here without going in and looking at what `add` is doing.